javers-persistence-mongo/src/main/java/org/javers/repository/mongo/MongoRepository.java

```diff
@@ -285,6 +326,17 @@ private Bson addVersionFilter(Bson query, Long version) {
         return Optional.of(readFromDBObject(dbObject));
     }
 
+    private List<CdoSnapshot> getSnapshots(Bson query) {
+        List<CdoSnapshot> snapshots = new ArrayList<>();
+        try (MongoCursor<Document> mongoSnapshots = getMongoSnapshotsCursor(query)) {
+            while (mongoSnapshots.hasNext()) {
```

Comment:
ten schemat się powtarza (zamiana `MongoCursor<Document>` na `List<CdoSnapshot>`), warto to wyciągnąć do metody